### PR TITLE
MAINT-27665: Fix the inability to edit news.

### PR DESCRIPTION
**Pull request by mdiha**
The issue is: when clicking on edit News button after the load of the page a message is displayed as if the user does not have the permissions.
In previous fixes we added the function canUserCreateNews() which is related to security and which needs the pace id to check whether the user has the permissions to add/edit news or not. But for the edit the space id property isn't considered so i made sure to reconsider this property in order to fix this issue.

### CODE REVIEW

File: webapp/src/main/webapp/news-details/components/ExoNewsActivityEditComposer.vue [hunk 1]
```diff
@@ -31,7 +35,7 @@ export default {
   },
   computed: {
     editLink: function() {
-      return `${eXo.env.portal.context}/${eXo.env.portal.portalName}/news/editor?newsId=${this.newsId}&activityId=${this.activityId}`;
+      return `${eXo.env.portal.context}/${eXo.env.portal.portalName}/news/editor?spaceId=${this.spaceId}&newsId=${this.newsId}&activityId=${this.activityId}`;
```
Comment (ahamdi) [inline webapp/src/main/webapp/news-details/components/ExoNewsActivityEditComposer.vue:38]: What if spaceId is null ?

Comment (mdiha) [reply webapp/src/main/webapp/news-details/components/ExoNewsActivityEditComposer.vue:38]: I didn't think of it. WDYT ?

Comment (mdiha) [reply webapp/src/main/webapp/news-details/components/ExoNewsActivityEditComposer.vue:38]: For now if space ID is null the user will be redirected to the page that show 'Permission denied ...'

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**